cli/cmd/lib_cli_config.go

```diff
@@ -139,27 +138,26 @@ func readCliConfig() (*CliConfig, []error) {
 
 	cliConfigData, err := cr.ReadJSONBytes(configBytes)
 	if err != nil {
-		cachedCliConfigErrs = []error{err}
-		return cachedCliConfig, cachedCliConfigErrs
+		cachedCLIConfigErrs = []error{err}
+		return cachedCLIConfig, cachedCLIConfigErrs
 	}
 
-	cachedCliConfigErrs = cr.Struct(cachedCliConfig, cliConfigData, fileValidation)
-	return cachedCliConfig, errors.WrapMultiple(cachedCliConfigErrs, configPath)
+	cachedCLIConfigErrs = cr.Struct(cachedCLIConfig, cliConfigData, fileValidation)
+	return cachedCLIConfig, errors.WrapMultiple(cachedCLIConfigErrs, configPath)
 }
 
-func getValidCliConfig() *CliConfig {
-	cliConfig, errs := readCliConfig()
+func getValidCLIConfig() *CLIConfig {
+	cliConfig, errs := readCLIConfig()
 	if len(errs) > 0 {
-		fmt.Printf("Environment \"%s\" is not configured, configuring now:\n", flagEnv)
```

Comment:
I think this should be added back as well. Also, might as well make it read a little better, e.g. 

```go
fmt.Printf("configuring %s environment:\n", flagEnv)
```